source/val/validate_decorations.cpp

```diff
@@ -1185,6 +1185,9 @@ spv_result_t CheckFPRoundingModeForShaders(ValidationState_t& vstate,
   // Validates Object operand of an OpStore
   for (const auto& use : inst.uses()) {
     const auto store = use.first;
+    if (store->opcode() == SpvOpFConvert) continue;
+    if (store->opcode() == SpvOpDecorate) continue;
```

Comment:
Should probably skip spvOpcodeIsDebug too. I would also consider adding spvOpcodeIsAnnotation and skipping all those instead of just the decorate and member decorate.

Reply:
Done.
FYI, there is no spvOpcodeIsAnnotation, I am assuming you meant spvOpcodeIsDecoration.